sportsdataverse/cfb/cfb_pbp.py

```
@@ -209,14 +209,39 @@ def _apply_wp_derivation(play_df, wp_before_raw, wp_touchback_raw, wp_after_raw,
     hwa = f"home_wp_after{suffix}"
     awa = f"away_wp_after{suffix}"
     wpa = f"wpa{suffix}"
+    # B5: the penalty-assessed-on-kickoff play takes the touchback wp_before too
+    # (0.36-live L5095), alongside the existing kickoff_vec substitution. The flag is
+    # only present on the full-pipeline frame; guard on its presence so the helper
+    # stays callable on a minimal synthetic frame (kickoff-only fallback).
+    touchback_mask = pl.col("type.text").is_in(kickoff_vec)
+    if "penalty_assessed_on_kickoff" in play_df.columns:
+        touchback_mask = touchback_mask.or_(pl.col("penalty_assessed_on_kickoff") == True)
```

Flake8 (7.3.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)